packages/server/modules/workspaces/graph/resolvers/workspaces.ts

```diff
@@ -483,6 +483,11 @@ export = FF_WORKSPACES_MODULE_ENABLED
           const workspacePlan = await getWorkspacePlanFactory({ db })({ workspaceId })
           if (workspacePlan) {
             switch (workspacePlan.name) {
+              case 'team':
+              case 'pro':
+                if (!FF_WORKSPACES_NEW_PLANS_ENABLED) {
```

Comment:
this if statement can be removed, team and pro is the same logic as starter plus and business